src/pretix/presale/templates/pretixpresale/event/base.html

```diff
@@ -31,7 +31,7 @@
             <div class="container">
                 <span class="fa fa-user-secret" aria-hidden="true"></span>
                 {% trans "This shop is currently only visible to you and your team." %}
-                <a href="{{ site_url }}{% url "control:event.live" event=event.slug organizer=event.organizer.slug %}">
+                <a href="{% url "control:event.live" event=event.slug organizer=event.organizer.slug %}">
```

Comment:
Avoid adding too many code into HTML attribute. You even made it worse by using the same quote style (") when nesting.

Please use `{% url 'some-url-name' arg arg2 as the_url %}` syntax to only insert a variable to HTML attribute.

Example: https://docs.djangoproject.com/en/5.1/ref/templates/builtins/#url